DB generator is generating code that doesn't comply with our static analysis rules

During https://github.com/lxc/lxd/pull/10600 @markylaing looks to have manually modified some of the `*.mapper.go` files rather than using `make update-schema` command in order to comply with the new static analysis rules.

See https://github.com/lxc/lxd/commit/430e5f33e44d211e084126adf4bf99d42e97cc0d

But this means now that when ever `make update-schema` is run, the `lxd/db/warnings.interface.mapper.go` and `lxd/db/warnings.mapper.go` are changed by the generator to a form that fails the static analysis tests.

Reproducer steps:

```
make update-schema
make static-analysis
golangci-lint run --timeout 5m
WARN [linters context] structcheck is disabled because of go1.18. You can track the evolution of the go1.18 support by following the https://github.com/golangci/golangci-lint/issues/2649. 
lxd/db/warnings.interface.mapper.go:5:69: Comment should end in a period (godot)
// WarningGenerated is an interface of generated methods for Warning
```

Instead of manually modifying the `*.mapper.go` files we need to update the DB generator to produce compliant code.

--- a/lxd/db/generate/db/method.go
+++ b/lxd/db/generate/db/method.go
@@ -464,12 +464,14 @@ func (m *Method) id(buf *file.Buffer) error {
 	buf.L("if !rows.Next() {")
 	buf.L(`        return -1, api.StatusErrorf(http.StatusNotFound, "%s not found")`, lex.Camel(m.entity))
 	buf.L("}")
+	buf.N()
 	buf.L("var id int64")
 	buf.L("err = rows.Scan(&id)")
 	m.ifErrNotNil(buf, "-1", "fmt.Errorf(\"Failed to scan ID: %w\", err)")
 	buf.L("if rows.Next() {")
 	buf.L("        return -1, fmt.Errorf(\"More than one row returned\")")
 	buf.L("}")
+	buf.N()
 	buf.L("err = rows.Err()")
 	m.ifErrNotNil(buf, "-1", "fmt.Errorf(\"Result set failure: %w\", err)")
 	buf.L("return id, nil")
@@ -503,6 +505,7 @@ func (m *Method) exists(buf *file.Buffer) error {
 	buf.L("        if api.StatusErrorCheck(err, http.StatusNotFound) {")
 	buf.L("                return false, nil")
 	buf.L("        }")
+	buf.N()
 	buf.L("        return false, err")
 	buf.L("}")
 	buf.N()
@@ -874,7 +877,7 @@ func (m *Method) signature(buf *file.Buffer, isInterface bool) error {
 
 	if isInterface {
 		buf.N()
-		buf.L("// %sGenerated is an interface of generated methods for %s", lex.Camel(m.entity), lex.Camel(m.entity))
+		buf.L("// %sGenerated is an interface of generated methods for %s.", lex.Camel(m.entity), lex.Camel(m.entity))
 		buf.L("type %sGenerated interface {", lex.Camel(m.entity))
 		defer m.end(buf)
 	}

